tests/braille-specs/sv.yaml

```diff
@@ -77,13 +71,16 @@ table:
   language: sv
   grade: 1
   __assert-match: sv-g1.ctb
+flags: {testmode: bothDirections}
+tests:
+  - # Non-contraction letters
+    - Nej s som i Sigurd.
+    - ⠠⠝⠑⠚ ⠒⠎ ⠎ ⠊ ⠠⠎⠊⠛⠥⠗⠙⠄
+flags: {testmode: forward}
```

Comment:
An alternative for running some tests only in forward direction is to use the [enhanced  `xfail` flag which now allows specifying the direction](https://github.com/liblouis/liblouis/issues/1223). I can have a look at this.

Reply:
Done